R/create_episode_file.R

```diff
@@ -437,7 +437,8 @@ join_sc_client <- function(data,
   if (file_type == "episode") {
     # Match on client variables by chi
     data_file <- data %>%
-      dplyr::left_join(sc_client,
+      dplyr::left_join(
+        sc_client %>% dplyr::select(-sc_send_lca),
```

Comment:
Great work finding the problem here. I have a suggestion that we could remove this from `process_lookup_sc_client` instead as this is the source. Other social care scripts create this which is why it appears in the bind rows. It would be lines 153-155 in the function `process_lookup_sc_client`. I think this would achieve the same as your suggestion here as in client processing we add this variable on but here we are just removing it. Any thoughts?